src/Okta.Sdk/Internal/DefaultDataStore.cs

```diff
@@ -140,6 +140,11 @@ private static void ApplyContextToRequest(HttpRequest request, RequestContext co
             {
                 request.Headers["Content-Transfer-Encoding"] = context.ContentTransferEncoding;
             }
+
+            if (!string.IsNullOrEmpty(context.ContentType))
+            {
+                request.ContentType = context.ContentType;
```

Comment:
What happens if contentType is null or empty? Do we have a default?

Reply:
The default content type is always "application/json".  This logic determines if that default should be overridden by what's specified in the request context.